protocol/messenger_contact_requests_test.go

```diff
@@ -1141,14 +1126,20 @@ func (s *MessengerContactRequestSuite) TestBobSendsContactRequestAfterDecliningO
 }
 
 func (s *MessengerContactRequestSuite) TestBuildContact() {
-	contactID := types.EncodeHex(crypto.FromECDSAPub(&s.m.identity.PublicKey))
-	contact, err := s.m.BuildContact(&requests.BuildContact{PublicKey: contactID})
+	bob := s.newMessenger(s.shh)
+	_, err := bob.Start()
```

Comment:
Actually, maybe a full started messenger is not needed for this test?
Looks like any generated PublicKey would be enough.

Reply:
fixed